packages/time-picker/test/value-commit.test.js

```diff
@@ -0,0 +1,391 @@
+        outsideClick();
+        expectValueCommit('12:00');
+      });
+
+      it('should revert and validate on close with Escape', async () => {
+        await sendKeys({ press: 'Escape' });
+        await sendKeys({ press: 'Escape' });
+        expectValidationOnly();
+        expect(timePicker.inputElement.value).to.equal(initialInputElementValue);
+      });
+    });
+
+    describe('entering unparsable input', () => {
+      beforeEach(async () => {
+        timePicker.inputElement.select();
+        await sendKeys({ type: 'INVALID' });
+      });
+
+      it('should commit an empty value on blur', async () => {
+        timePicker.blur();
+        expectValueCommit('');
+        expect(timePicker.inputElement.value).to.equal('INVALID');
+      });
+
+      it('should commit an empty value on Enter', async () => {
+        await sendKeys({ press: 'Enter' });
+        expectValueCommit('');
+        expect(timePicker.inputElement.value).to.equal('INVALID');
+      });
+
+      it('should commit an empty value on close with outside click', () => {
+        outsideClick();
+        expectValueCommit('');
+        expect(timePicker.inputElement.value).to.equal('INVALID');
+      });
+
+      it('should revert and validate on close with Escape', async () => {
+        await sendKeys({ press: 'Escape' });
+        expectValidationOnly();
+        expect(timePicker.inputElement.value).to.equal(initialInputElementValue);
+      });
+    });
+
+    describe('clearing input with Backspace', () => {
+      beforeEach(async () => {
+        timePicker.inputElement.select();
+        await sendKeys({ press: 'Backspace' });
+      });
+
+      it('should commit on blur after clearing', () => {
+        timePicker.blur();
+        expectValueCommit('');
+      });
+
+      it('should commit on Enter after clearing', async () => {
+        await sendKeys({ press: 'Enter' });
+        expectValueCommit('');
+      });
+
+      it('should commit on outside click after clearing', async () => {
+        outsideClick();
+        expectValueCommit('');
+      });
+
+      it('should revert and validate on close with Escape after clearing', async () => {
+        await sendKeys({ press: 'Escape' });
+        expectValidationOnly();
+        expect(timePicker.inputElement.value).to.equal(initialInputElementValue);
+      });
+    });
+
+    describe('with clear button', () => {
+      beforeEach(() => {
+        timePicker.clearButtonVisible = true;
+      });
+
+      it('should clear on clear button click', () => {
+        timePicker.$.clearButton.click();
+        expectValueCommit('');
+      });
+
+      it('should clear on Escape', async () => {
+        await sendKeys({ press: 'Escape' });
+        expectValueCommit('');
+      });
+    });
+  });
+
+  describe('with step', () => {
+    beforeEach(() => {
+      timePicker.step = 1;
+    });
+
+    it('should commit on ArrowDown', async () => {
+      await sendKeys({ press: 'ArrowUp' });
```

Comment:
The key does not match the test name:

```suggestion
      await sendKeys({ press: 'ArrowDown' });
```

Reply:
Fixed.